scipy/cluster/hierarchy.py

```diff
@@ -457,7 +454,9 @@ def ward(y):
     Returns
     -------
     Z : ndarray
-        The hierarchical clustering encoded as a linkage matrix.
+        The hierarchical clustering encoded as a linkage matrix. See
+       linkage for more information on the return structure and
+       algorithm.
```

Comment:
The indentation of the above 2 lines is incorrect, add one space. Also, add single backticks around ``linkage`` so it becomes a link.